guides/introduction/templates.md

```diff
@@ -0,0 +1,202 @@
+# Template
+
+We call template to almost every file in the input folder. These templates
+are transformed, and combined, to create the website.
+
+## Helpers
+
+In templates, you can usually write Elixir, which allows you to extend
+your site with abstractions to generate CSS or make API calls. Still comes
+with some functions out of the box to help you build website. We call
+these view helpers.
+
+### Including other files
+
+Use the `include` function to import the contents of another file. For
+instance, the following template includes the file `_includes/head.slime`:
+
+```slim
+html
+  head
+    = include "_includes/head.slime"
+  body
+    = children
+```
+
+Behind the scenes, Still will render `head.slime` to HTML and replace the
+function call with its contents.
+
+### Linking to other files
+
+The `link` function creates HTML hyperlinks. It takes care of specifying
+the `rel` and `target` when necessary, and it supports both full URLs or
+paths relative to the input folder. You should always use the link
+function to create hyperlinks between files, otherwise deploys to
+subfolders will not work. For instance:
+
+```slim
+= link "Home", to: "/"
+= link "Blog", to: "https://example.org"
+```
+
+### Including images
+
+Image support is still (get it?) rudimentary: it can generate images of
+different sizes, and apply transformations to them. By default, it relies
+on [ImageMagick][imagemagick], but if you add [`imageflow_ex`][imageflow]
+as a dependency it will use that. The reason we do not use `imageflow_ex`
+by default is that it requires Rust. If you want to use ImageMagick, you
+also need to have it installed in your system.
+
+`responsive_image` receives an image and some opts and generates an HTML
+`img` which with the correct `src` and `sizes` for the different image
+sizes it generates.
+
+To insert an image use the `responsive_image` function, which accepts
+a list of keyword arguments. To generate an image, Still only cares about the
+`:image_opts` key, everything else will be rendered as an attribute on the
+`img` tag. `:image_opts` has to be a map where you can specify `:sizes`
+and `:transformations`.
```

Comment:
This paragraph should replace the previous one, right? And use `responsive_image/2` instead, please 🙏